res/values-it/string.xml

```diff
@@ -162,5 +162,11 @@
 
     <!-- DoubleTap2Sleep -->
     <string name="double_tap_sleep_lockscreen_title">Doppio tap per spegnere lo schermo</string>
-    <string name="double_tap_sleep_lockscreen_sumary">Doppio tap in qualsiasi parte della schermata di blocco per spegnere lo schermo</string>
+    <string name="double_tap_sleep_lockscreen_summary">Doppio tap in qualsiasi parte della schermata di blocco per spegnere lo schermo</string>
```

Comment:
Also it is necessary to change in those places where there is a reference to this line 
(example lockscreen_settings.xml)